homeassistant/components/scrape/sensor.py

```diff
@@ -99,27 +100,22 @@ async def async_setup_platform(
 
         coordinator = ScrapeCoordinator(hass, rest, SCAN_INTERVAL)
 
-        sensors_config: list[tuple[ConfigType, ConfigType]] = [
-            (
-                config,
-                vol.Schema(TEMPLATE_SENSOR_BASE_SCHEMA.schema, extra=vol.REMOVE_EXTRA)(
-                    config
-                ),
+        sensors_config = [
+            vol.Schema(TEMPLATE_SENSOR_BASE_SCHEMA.schema, extra=vol.ALLOW_EXTRA)(
```

Comment:
By using ALLOW_EXTRA for the legacy platform, we don't need to carry over a split config for `template` and `sensor`.